src/sounds.cpp

```diff
@@ -388,16 +393,790 @@ std::pair<std::vector<tripoint>, std::vector<tripoint>> sounds::get_monster_soun
     return { sound_locations, cluster_centroids };
 }
 
-std::string sounds::sound_at( const tripoint &location )
-{
+std::string sounds::sound_at( const tripoint &location ) {
     auto this_sound = sound_markers.find( location );
     if( this_sound == sound_markers.end() ) {
         return std::string();
     }
-
     if( !this_sound->second.description.empty() ) {
         return this_sound->second.description;
     }
+    return _( "a sound" );
+}
+
+//sfx::
+void sfx::load_sound_effects( JsonObject &jsobj ) {
+    set_group_channels( 2, 9, 1 );
+    set_group_channels( 0, 1, 2 );
+    set_group_channels( 11, 14, 3 );
+    set_group_channels( 15, 17, 4 );
```

Comment:
Why is this called every time an sound effect is loaded? It does not seem to affect the loaded effect, it should probably only be called once, in some startup code - maybe after `Mix_OpenAudio` in `WinCreate` in sdltiles.cpp?